custom_components/afvalbeheer/sensor.py

```diff
@@ -240,6 +242,7 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
     sensor_today = config.get(CONF_TODAY_TOMORROW)
     date_object = config.get(CONF_DATE_OBJECT)
     name = config.get(CONF_NAME)
+    language = config.get(CONF_LANGUAGE)
```

Comment:
We should deprecate `dutch_days` and combine it with language. Maybe for now still allow the option `dutch_days` and set language = NL if this option is present to be more backwards compatible

Reply:
Thx for your feedback @pippyn .
That is also for backward compatibility that I used
`vol.Optional(CONF_LANGUAGE, default='nl'): cv.string,`
As I guess most of the users are NL speakers (but I actually did not test it... only with 'fr') 
> Maybe for now still allow the option dutch_days and set ...

Do you mean you would prefer this to be implemented (and a translation dict) before merging this PR?

Reply:
Yes indeed. That would make the code cleaner and more consistent.

Reply:
BTW: I have tests in place to test the code for Dutch users